src/components/Product.js

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+
+const Article = styled.article`
+  width: 100%;
+  display: flex;
+  -webkit-box-pack: justify;
+  justify-content: space-between;
+  font-size: 1em;
+  color: black;
+  cursor: pointer;
+  flex-direction: column;
+`;
+
+const ImageContainer = styled.div`
+  height: 150px;
+  display: flex;
+  -webkit-box-pack: center;
+  justify-content: center;
+`;
+
+const Image = styled.img`
+  max-height: 100%;
+  max-width: 100%;
+`;
+
+const Details = styled.div`
+  align-self: center;
+  width: 100%;
```

Comment:
given the parent of this div is flex, this div by default uses `100%`, by adding the left and right padding, it may give the sense that is not working, for example you could remove this `width: 100%`, and instead make the padding to the left and right 0 like: `padding: 16px 0;`

Something similar will happen to the `Button` inside this `Details` container, if `Details` becomes `flex`, then the button should no need to have a of `width: 100%`